lib/js/jsbigints.nim

```diff
@@ -155,12 +155,40 @@ func inc*(x: var JsBigInt) {.importjs: "(++[#][0][0])".} =
     inc big1
     doAssert big1 == big"2"
 
-func dec*(x: var JsBigInt;) {.importjs: "(--[#][0][0])".} =
+func dec*(x: var JsBigInt) {.importjs: "(--[#][0][0])".} =
   runnableExamples:
     var big1: JsBigInt = big"3"
     dec big1
     doAssert big1 == big"2"
 
+func `%=`*(x: var JsBigInt; y: JsBigInt) {.importjs: "([#][0][0] %= #)".} =
```

Comment:
also deserves a doc comment because of mismatch bw mod and %=:
```nim
## Same as: `x = x mod y`
```

ditto for /=:
```nim
## Same as: `x = x div y`
```